src/parser/expression.js

```diff
@@ -300,7 +300,8 @@ pp.parseSubscripts = function (base, startPos, startLoc, noCalls) {
       let node = this.startNodeAt(startPos, startLoc);
       node.callee = base;
       node.arguments = this.parseCallExpressionArguments(tt.parenR, possibleAsync);
-      base = this.finishNode(node, "CallExpression");
+      base = this.finishNode(node, this.state.inImportCall ? "ImportCallExpression" : "CallExpression");
```

Comment:
@loganfsmyth Alright I updated the code to do that. Ended up being a pretty clean solution! Thanks for the recommendation.